.mise.toml

```
@@ -6,7 +6,7 @@ KOTLIN_POST_PROCESS_FILE = "ktlint -F"
 node = '22.10.0'
 "npm:prettier" = "3.3.3"
 maven = "3.9.9"
-awscli = "ref:2.18.6"
+awscli = "2.18.9"
```

Laitetaan ref takaisin, kun se on määritetty https://github.com/Opetushallitus/koto-rekisteri/pull/229 tehtävässä

```suggestion
awscli = "ref:2.18.9"
```